app/controllers/cart_staches_controller.rb

```diff
@@ -10,6 +11,7 @@ def destroy
     stache = Stache.find(params[:id])
     @cart.delete_stache(stache.id)
     session[:cart] = @cart.contents
-    redirect_to cart_path
+    flash.now[:removed] = "Successfully removed #{view_context.link_to stache.name, stache_path(stache)} from your cart.".html_safe
+    render 'carts/show'# redirect_to cart_path
```

Comment:
Put a space before an end-of-line comment.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.